packages/app/src/components/tag/TagStepImage.vue

```diff
@@ -77,7 +81,7 @@ export default defineComponent({
     });
 
     onBeforeMount(async () => {
-      const track = await feathers.service('spotify-tracks').get(props.nfcTag.trackUri);
+      const track = await getTrackOfTag(props.nfcTag);
```

Comment:
nice to see that my approch is used :D :D

Reply:
😉